src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/CaseSearchRequest.java

```diff
@@ -30,13 +29,6 @@
     public CaseSearchRequest(String caseTypeId, JsonNode searchRequestJsonNode) {
         this.caseTypeId = caseTypeId;
         this.searchRequestJsonNode = searchRequestJsonNode;
-        validateJsonSearchRequest();
-    }
-
-    private void validateJsonSearchRequest() {
```

Comment:
it wouldn't hurt to leave this isn't it?
If we remove it and then for some reason if the future we use this outside the multi search, then we don't have the validation

Reply:
yes restored